ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala

```diff
@@ -121,9 +140,11 @@ private[kvutils] class PackageCommitter(
     })
   }
 
-  private val enqueuePreload: Step = (_, uploadEntry) => {
-    preloadExecutor.execute(
-      preload(uploadEntry.getSubmissionId, uploadEntry.getArchivesList.asScala))
+  private[this] val enqueueFullValidation: Step = { (_, uploadEntry) =>
```

Comment:
Why the name change to `enqueueFullValidation`? It isn't really validating them but rather preloading them since failures are not propagated anywhere.